interface-elements-for-web/articles/grid/customization-dialog/sorting-page.md

```diff
@@ -4,11 +4,21 @@ author: Anna Kondratova
 legacyId: 118811
 ---
 # Sorting Page
-Select a column from the "Columns to Sort" list to sort grid data against this column in ascending order. The grid can be sorted more than by one column, and a list of the sorted columns is displayed at the top of the **Sorting** page.
+The **Sorting** page enables you to sort grid data, change the sorting order, and change the order of sorted columns within the grid.
+
+Select a column's check box in the **Columns to Sort** list, to sort grid data against this column in ascending order. The grid can be sorted more than by one column; a list of the sorted columns is displayed at the top of the **Sorting** page.
```

Comment:
"list, to" Again, why do you use a comma here?

Reply:
"The grid can be sorted more than by one column; a list of the sorted columns is displayed at the top of the **Sorting** page." Why do you use a semicolon here?

Reply:
Perhaps it is better to split this sentence into two sentences?
"The grid can be sorted more than by one column. A list of the sorted columns is displayed at the top of the Sorting page."